saleor/plugins/admin_email/notify_events.py

```diff
@@ -24,6 +24,9 @@ def send_set_staff_password_email(
         constants.SET_STAFF_PASSWORD_DEFAULT_TEMPLATE,
         constants.DEFAULT_EMAIL_TEMPLATES_PATH,
     )
+    if not template:
+        # Empty template means that we don't want to trigger a given event.
```

Comment:
How about to create a `@decorator` for this statement? Not in this scope but in the future as an enhancement?